### DISCUSSION TITLE
mxSession.invitedRooms() always returns nil

### DISCUSSION THREAD

**Discussion by vraj-s-shah** [element-hq/element-meta#1685] [Q&A]
Recently, I was trying to get all invited rooms for the user so I can accept and join the room but the method `invitedRooms()` of MXSession always returns nil. Has anyone tried this already and managed to get invited rooms?

**Comment by t3chguy** [comment]
@vraj-s-shah wrong repo, open an issue in https://github.com/matrix-org/matrix-ios-sdk

**Reply by vraj-s-shah** [reply]
Yes, I have already opened an issue over there but haven't received a response so I tried here also just so if someone might have a solution.

**Reply by t3chguy** [reply]
These discussions aren't about any specific SDK, they're about high-level Element product/feature requests
